dev-cpp/cpp-httplib/cpp-httplib-0.11.3.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..11} )
```

Comment:
It doesn't build with 3.8 or 3.9?

Reply:
It does, added. Do we need to care about old python for new packages?

Reply:
> It does, added. Do we need to care about old python for new packages?

Well it probably doesn't really matter for this package, since the python dependency is build-time only. But we usually drop a python implementation throughout the entire tree in one go to avoid making a dependency mess. Older systems could still be using 3.9, we still support this.